### ISSUE TITLE
Section Indicator on page created in 3.7 breaks ability to add new CTP after upgrading to 3.8 with Guttenberg Integration enabled

### ISSUE THREAD

**Issue by KirbiDev** [soflyy/oxygen-bugs-and-features#2260]
Description

Site using Metabox CPTs and Guttenberg Integration generates a uncaught error when adding a new CPT after upgrading to 3.8

The error generated is 

```
2021/06/22 16:10:43 [error] 27743#27743: *4343 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to a member function global_conditions_result() on null in /var/www/mysite.xyzwp.com/htdocs/wp-content/plugins/oxygen/component-framework/components/component.class.php:3843
Stack trace:
#0 /var/www/mysite.xyzwp.com/htdocs/wp-content/plugins/oxygen/component-framework/components/classes/div-block.class.php(39): CT_Component->validate_shortcode()
#1 /var/www/mysite.xyzwp.com/htdocs/wp-includes/shortcodes.php(343): CT_DIV_Block->add_shortcode()
#2 [internal function]: do_shortcode_tag()
#3 /var/www/mysite.xyzwp.com/htdocs/wp-includes/shortcodes.php(218): preg_replace_callback()
#4 /var/www/mysite.xyzwp.com/htdocs/wp-content/plugins/oxygen-gutenberg/oxygen-gutenberg.php(179): do_shortcode()
#5 /var/www/mysite.xyzwp.com/htdocs/wp-includes/class-wp-hook.php(292): Oxygen_Gutenberg->current_screen()
#6 /var/www/mysite.xyzwp.com/htdocs/wp-includes/class-wp-hook.php(316): WP_Hook->apply_filters()
#7 /var/www/mysite.xyzwp.com/htdocs" while reading response header from upstream, client: 117.20.66.111, server: mysite.xyzwp.com, request: "GET /wp-admin/post-new.php?post_type=service HTTP/2.0", upstream: "fastcgi://unix:/var/run/php/php74-fpm-mysite.xyzwp.com.sock:", host: "mysite.xyzwp.com", referrer: "https://mysite.xyzwp.com/wp-admin/edit.php?post_type=page"
```

Cause

At first I thought this had something to do with the Oxygen Guttenberg Integration because disabling it fixed the problem. I then tried reproducing it in a clean site but could not. 

I then started migrating, different elements at a time ( post_types, taxonomies, CPT posts, pages, etc.. ) to the test site and kept testing until it broke. Turns out it was a page causing the problem. 

I deleted all the pages and then put them back one at a time until I discovered it was the home page causing the problem. I then deleted sections to see if I could narrow down where the problem was. Only thing left was the Section Indicator. I removed it and we were back in business. I then restored the page to it's original form, tested again and it was broken. This time I just removed the section indicator and it was fixed. 

I then discovered that if I re-added the section indicator (after the 3.8 upgrade) everything worked. So it seems, the upgrade either breaks something in a 3.7 section indicator or doesn't upgrade something it should in that component.

To reproduce

1. Create a site with all the basics: Oxygen 3.7.1, Guttenberg Integration enabled, MetaBox.io plugin
2. Create the necessary templates 
3. create a page with a couple of sections and add a section indicator (I has the page as homepage - it might be relevant)
4. add a CPT (use any name you like)
5. click "add new" for the CPT and everything should work

upgrade Oxygen to 3.8 

1. click "add new" for CPT and it should throw the error (as above)
2. disable Guttenberg Integration and the error will disappear
3. re-enable it and the error comes back
4. edit in Oxygen the page with the section indicator and remove the section indicator and save 
5. error should be gone
6. re-add the section indicator
7. error still gone

An obscure one but might be causing other issues?

**Comment by KittenCodes** [comment]
@KirbiDev Please open a support ticket so that we can take a look at this for you: https://oxygenbuilder.com/support/.

**Comment by KirbiDev** [comment]
I don't need you to take a look at it. I was just reporting it so you guys were aware of it. The work-around I found (removing the Section Indicator originally created in 3.7.1 and then re-adding it after the upgrade to 3.8) works fine. Obviously there is something broken in the upgrade process that you guys aren't aware of (because I can recreate this issue on a fresh install using the process I described above). I have a work around now and I'm not interested in spending any more time on it. Wasted half a day already just tracking it down.
